<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkValidator</class>
 <widget class="QMainWindow" name="LinkValidator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.625, y1:0.744, x2:1, y2:0, stop:0 rgba(55, 55, 55, 255), stop:1 rgba(37, 37, 37, 255));
color: rgb(240, 240, 240);
border-radius: 5%;
font: 10pt &quot;Arial&quot;;

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowminimumheight="0,0,0,0,40">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="4" column="1" colspan="2">
     <widget class="QPushButton" name="stopValidateButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 8pt &quot;Arial Black&quot;;
	
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.472, y1:0.585, x2:1, y2:0, stop:0.00568182 rgba(250, 167, 1, 255), stop:1 rgba(234, 128, 7, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 10pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
      </property>
      <property name="text">
       <string>Stop validate</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QWidget" name="Header" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.472, y1:0.585, x2:1, y2:0, stop:0.00568182 rgba(250, 167, 1, 255), stop:1 rgba(234, 128, 7, 255));</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,2,9,2,2,2,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="imgLogo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>image/linkvalidlogo.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="textLogo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
font: 87 14pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Link Validator</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="manualButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 8pt &quot;Arial Black&quot;;
	
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
	border-radius: 50%;
	color: rgb(255, 255, 255);
	font: 87 10pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
         </property>
         <property name="text">
          <string>  Manual  </string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="faqButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 8pt &quot;Arial Black&quot;;
	
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
	border-radius: 50%;
	color: rgb(255, 255, 255);
	font: 87 10pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
         </property>
         <property name="text">
          <string>FAQ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="copyButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 20pt &quot;Arial Black&quot;;
	
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
	border-radius: 50%;
	color: rgb(255, 170, 0);
	font: 87 24pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
         </property>
         <property name="text">
          <string>‚ùê</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>textLogo</zorder>
      <zorder>manualButton</zorder>
      <zorder>faqButton</zorder>
      <zorder>horizontalSpacer</zorder>
      <zorder>horizontalSpacer_3</zorder>
      <zorder>horizontalSpacer_7</zorder>
      <zorder>imgLogo</zorder>
      <zorder>copyButton</zorder>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QWidget" name="Success" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
border-radius: 5%;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="successLinks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>2xx: Success</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="successEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
font: 87 16pt &quot;Arial Black&quot;;
border-radius: 5%;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QWidget" name="Reidrect" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
border-radius: 5%;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="redirectLinks">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>3xx: Redirection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="redirectEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
font: 87 16pt &quot;Arial Black&quot;;
border-radius: 5%;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="Error" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
border-radius: 5%;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="errorLinks">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>4xx: Client Error</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="errorEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
font: 87 16pt &quot;Arial Black&quot;;
border-radius: 5%;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QWidget" name="Total" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
border-radius: 5%;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="totalLinks">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
color: rgb(255, 170, 0);
font: 87 10pt &quot;Arial Black&quot;;</string>
         </property>
         <property name="text">
          <string>Total links</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="totalEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
font: 87 16pt &quot;Arial Black&quot;;
border-radius: 5%;</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="4">
     <widget class="QWidget" name="Sidebar" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
border-radius: 5%;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="2,2,2,14,0,1">
       <property name="leftMargin">
        <number>7</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>7</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QPushButton" name="checkLinkButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.472, y1:0.585, x2:1, y2:0, stop:0.00568182 rgba(250, 167, 1, 255), stop:1 rgba(234, 128, 7, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 9pt &quot;Arial Black&quot;;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.329545, y1:0.739, x2:1, y2:0, stop:0 rgba(255, 216, 57, 255), stop:1 rgba(255, 173, 0, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 10pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
         </property>
         <property name="text">
          <string>Check Link</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gDocsButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.472, y1:0.585, x2:1, y2:0, stop:0.00568182 rgba(250, 167, 1, 255), stop:1 rgba(234, 128, 7, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 9pt &quot;Arial Black&quot;;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.329545, y1:0.739, x2:1, y2:0, stop:0 rgba(255, 216, 57, 255), stop:1 rgba(255, 173, 0, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 10pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
         </property>
         <property name="text">
          <string>Google Docs report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="excelButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0.472, y1:0.585, x2:1, y2:0, stop:0.00568182 rgba(250, 167, 1, 255), stop:1 rgba(234, 128, 7, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 9pt &quot;Arial Black&quot;;
}

QPushButton:hover {
	background-color: qlineargradient(spread:pad, x1:0.329545, y1:0.739, x2:1, y2:0, stop:0 rgba(255, 216, 57, 255), stop:1 rgba(255, 173, 0, 255));
	border-radius: 5%;
	color: rgb(255, 255, 255);
	font: 87 10pt &quot;Arial Black&quot;;
	transition: 0.3s ease;
}</string>
         </property>
         <property name="text">
          <string>Excel report</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="styleSheet">
          <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
border-radius: 5%;
font: 10pt &quot;Ink Free&quot;;</string>
         </property>
         <property name="text">
          <string>version  1.0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="designButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 0), stop:1 rgba(255, 255, 255, 0));
	color: rgb(240, 240, 240);
	font: 12pt &quot;Ink Free&quot;;
	border-radius: 5%;
}

QPushButton:hover {
	color: rgb(255, 170, 0);
}</string>
         </property>
         <property name="text">
          <string>Designed by Eugene</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QPlainTextEdit" name="logMonitor">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0.272727, y1:0.767, x2:1, y2:0, stop:0 rgba(81, 81, 81, 255), stop:1 rgba(100, 100, 100, 255));
border-radius: 5%;
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
